packages/components-js/tests/vrt/puppeteer/specs/table.vrt.ts

```diff
@@ -53,16 +54,49 @@ xit('should have no visual regression for :hover + :focus-visible', async () =>
 
       // TODO: scroll trigger :hover + :focus-visible test is missing due piercing selector only works for nested child
       // TODO: `await forceFocusedState(page, '.focus p-table >>> .scroll-area');`, no class is selectable after piercing selector
-      await forceHoverState(page, '.hover p-table-head-cell >>> button');
+
+      await forceFocusState(page, '.focus p-table-cell a');
+      await forceFocusState(page, '.focus [slot="caption"] a');
+      await forceFocusState(page, '.focus p-table-head-cell >>> p-button-pure >>> button');
+
+      await forceFocusHoverState(page, '.focus-hover p-table-cell a');
+      await forceFocusHoverState(page, '.focus-hover [slot="caption"] a');
+      await forceFocusHoverState(page, '.focus-hover p-table-head-cell >>> p-button-pure >>> button');
+
       await forceHoverState(page, '.hover p-table-cell a');
       await forceHoverState(page, '.hover [slot="caption"] a');
       await forceHoverState(page, '.hover p-table-row:nth-child(3)');
-      await forceFocusState(page, '.focus p-table-head-cell >>> button');
+      await forceHoverState(page, '.hover p-table-head-cell >>> p-button-pure >>> button');
+    })
+  ).toBeFalsy();
+});
+
+it('should have no visual regression for :hover + :focus-visible and dark theme', async () => {
```

Comment:
Hmm, not sure why we now all of a sudden split the states tests into separate fixture.  
In any case, having an `it.each` with both themes would be less redundant and easier to maintain than the current approach.

Reply:
@denyo The reason for that was failing vrt test in pipeline (sometimes). I discussed it in daily and we decided that I split light and dark into separate tests, and it helped. It's not optimal, but it was for me a way to make it work most of the time (instead of failing almost all the time when light & dark were in one test)